src/canvas.py

```diff
@@ -293,7 +313,7 @@ def right_scale(self, scale: int):
 
     @GObject.Property(type=float)
     def min_bottom(self):
-        return min(self.axis.get_xlim())
+        return self.axis.get_xlim()[0]
```

Comment:
In principle the first element should always be equal to the minimum value, so both sides are equivalent. In my view using `min` or `max` of the limits is slightly more readable though. Any specific reason for changing these?

Reply:
My main reason was performance gain, as we don't have to compare the two values to see which is the smaller. 
According to [matplotlib documentation](https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.set_xlim.html), they represent left and right (or bottom and top for y), which matches our usage as upper/lower